File: cfgs/java_cfgs_foo/COUNT_ENTRIES_EQUAL_TO_X_IN_A_SPECIAL_MATRIX.java
Nodes
1;special;;;
2;statement;int f_gold = 0;14;
3;statement;int i = 1;15;
4;conditional;(foo_1 <= foo_2) && (foo_3 <= foo_4);16;
5;conditional;((foo_1 / foo_2) <= foo_3) && ((foo_4 % foo_5) == 0);18;
6;statement;f_gold++;18;
7;statement;i++;17;
8;statement;return foo_1;20;
9;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;8;
4;6;7;
5;7;
6;4;
7;9;
8;

File: cfgs/pythoncfgs/COUNT_ENTRIES_EQUAL_TO_X_IN_A_SPECIAL_MATRIX.py
Nodes
1;statement;foo_1 = 0;8;
2;conditional;foo < foo;9;
3;statement;return foo_1;13;
4;conditional;foo_1 <= foo_2;10;
5;conditional;foo_1 // foo_2 <= foo_3 and foo_4 % foo_5 == 0;11;
6;statement;foo_1 += 1;12;
7;exit;;
Edges
0;2;
1;3;4;
2;7;
3;2;5;
4;2;6;
5;2;
6;
